include/picongpu/fields/incidentField/profiles/DispersivePulse.hpp

```diff
@@ -299,15 +299,39 @@ namespace picongpu
                             float_X const dt = static_cast<float_X>(picongpu::SI::DELTA_T_SI / UNIT_TIME);
                             // interpolation order
                             float_X N_raw = Unitless::INIT_TIME / dt;
-                            int n = static_cast<int>(N_raw * 0.5_X); // -0 instead of -1 for rounding up N_raw
+                            int const n = static_cast<int>(N_raw * 0.5_X); // -0 instead of -1 for rounding up N_raw
+
+                            // frequency step for DFT
+                            float_X const dOmk = pmacc::math::Pi<float_X>::doubleValue / Unitless::INIT_TIME;
+
+                            // Since the (Gaussian) spectrum has only significant values near the central frequency,
+                            // the summation over all frequencies is reduced to frequencies within an interval
+                            // around the central frequency.
+
+                            // standard deviation of the Gaussian distributed spectrum
+                            float_X const sigma_Om = 1._X / (pmacc::math::sqrt(2._X) * Unitless::PULSE_DURATION);
```

Comment:
Where does this come from? Should this not be `2 * pi * 0.441 / Unitless::Unitless::PULSE_DURATION` for a bandwidth limited pulse?

Reply:
It should be right, it is exactly the standard deviation used a few lines above to define the longitudinal (gaussian) envelope of the pulse in the frequency domain.

Reply:
Hm, it's some different definition of pulse duration here than in the shadowgraphy plugin or in my post-processing. I can't find this right now, but if this here is self-consistent and producing pulses with the correct duration it should be right